composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -106,7 +106,16 @@ data class Args(
                 description = "Either `true` or `false` to enable/disable error on empty test suite. True by default.",
                 order = 11
         )
-        var failIfNoTests: Boolean = true
+        var failIfNoTests: Boolean = true,
+
+        @Parameter(
+                names = arrayOf("--coverage"),
+                required = false,
+                arity = 1,
+                description = "Either `true` or `false` to enable/disable test coverage collection. `false` by default.",
```

Comment:
Should we add a note that it doesn't turn on code coverage in the tests but just collects it?

README needs to be updated too btw